docs/root/intro/version_history.rst

```diff
@@ -6,6 +6,7 @@ Version history
 * admin: added support for displaying subject alternate names in :ref:`certs<operations_admin_interface_certs>` end point.
 * config: removed support for the v1 API.
 * fault: removed integer percentage support.
+* http: Added HTTP/2 WebSocket proxying via :ref:`extended CONNECT <envoy_api_field_core.Http2ProtocolOptions.allow_connect>`
```

Comment:
Related to my comment on the old style removal PR, but do we need any stats/observability for this stuff? If so it's fine to track as a follow up but might want to put in a TODO somewhere?